package.json

```diff
@@ -1,13 +1,9 @@
 {
 	"name": "faucet-pipeline-images",
-	"version": "0.0.5",
+	"version": "0.1.0",
 	"description": "Images Asset Pipeline",
-	"main": "index.js",
 	"scripts": {
-		"test": "npm-run-all --parallel lint test:cli",
-		"test:cli": "./test/run",
-		"test:prepare": "./test/prepare",
-		"lint": "eslint --cache index.js test && echo ✓"
+		"test": "echo 'No Code, No Tests'"
```

Comment:
Why even have this line in the first place then? Indeed, it's misleading if you only check the exit code.

(`release` would need to be adjusted accordingly.)